{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/__init__.py

```diff
@@ -1,7 +1,7 @@
 from .version import version as __version__
 
-{%- if cookiecutter.include_example_code == 'y' %}
 from .example_mod import do_primes
+{%- if cookiecutter.include_example_code == 'y' %}
 # Then you can be explicit to control what ends up in the namespace,
```

Comment:
Going to need to make sure isort doesnt move this with # isort:skip ?

Reply:
yeah this is what i'm trying to sort atm :disappointed: